File: cases/taint/typescript/local-overwrite-negative/LocalOverwriteNegative.ts
function dfb_source(): string { // DFB-SOURCE: local-overwrite-input
  return "tainted";
}

function dfb_sink(value: string): void {} // DFB-SINK: local-overwrite-sink

function run(): void {
  let value: string = dfb_source();
  value = "clean"; // DFB-KILL: local-overwrite-clean
  dfb_sink(value);
}

File: cases/taint/rust/local-overwrite-negative/local_overwrite_negative.rs
fn dfb_source() -> i32 { // DFB-SOURCE: local-overwrite-input
    1
}

fn dfb_sink(value: i32) {} // DFB-SINK: local-overwrite-sink

fn run() {
    let mut value = dfb_source();
    value = 0; // DFB-KILL: local-overwrite-clean
    dfb_sink(value);
}
